package.json

```diff
@@ -49,6 +49,7 @@
     "cssom": "^0.3.0",
     "domhandler": "^2.3.0",
     "escodegen": "^1.4.1",
+    "eslint": "^1.10.3",
```

Comment:
Or we might just use separate repo with purely parser API and deps, which will be referenced as a dependency by UI.